lib/manager/components/deployment/DeploymentSettings.js

```diff
@@ -206,111 +205,116 @@ class DeploymentSettings extends Component<Props, State> {
-              defaultExpanded={!u.type}
-              onRemove={this._onRemoveUpdater}
-              onChange={this._getOnChange}
-              testId={u.type
-                ? `${u.type}-${i}`
-                : `[${this.messages('routerConfig.updaters.placeholder')}]`
-              }
-              title={u.type
-                ? <span>
-                  {u.type}{'  '}
-                  <small>{u.url}</small>
-                </span>
-                : `[${this.messages('routerConfig.updaters.placeholder')}]`
-              }
-            />
-          ))}
+          </Panel.Title></Panel.Heading>
+          <Panel.Body>
+            {updaters.map((u, i) => (
+              <CollapsiblePanel
+                data={u}
+                defaultExpanded={!u.type}
+                fields={UPDATER_FIELDS}
+                index={i}
+                key={i}
+                onChange={this._getOnChange}
+                onRemove={this._onRemoveUpdater}
+                testId={u.type
+                  ? `${u.type}-${i}`
+                  : `[${this.messages('routerConfig.updaters.placeholder')}]`
+                }
+                title={u.type
+                  ? <span>
+                    {u.type}{'  '}
+                    <small>{u.url}</small>
+                  </span>
+                  : `[${this.messages('routerConfig.updaters.placeholder')}]`
+                }
+              />
+            ))}
+          </Panel.Body>
         </Panel>
         {/* OSM extract settings */}
-        <Panel header={<h4><Icon type='globe' /> {this.messages('osm.title')}</h4>}>
-          <FormGroup
-            onChange={this._onToggleCustomBounds}>
-            <Radio
-              checked={!this._getValue('useCustomOsmBounds')}
-              name='osm-extract'
-              value={false}>
-              {this.messages('osm.gtfs')}
-            </Radio>
-            <Radio
-              checked={this._getValue('useCustomOsmBounds')}
-              name='osm-extract'
-              value>
-              {this.messages('osm.custom')}
-            </Radio>
-          </FormGroup>
-          {project.useCustomOsmBounds || this.state.useCustomOsmBounds
-            ? <FormGroup>
-              <ControlLabel>
-                <span>
-                  <Glyphicon glyph='fullscreen' /> {this.messages('osm.bounds')}
-                </span>
-              </ControlLabel>
-              <FormControl
-                type='text'
-                defaultValue={project.bounds
-                  ? `${project.bounds.west},${project.bounds.south},${project.bounds.east},${project.bounds.north}`
-                  : ''
-                }
-                placeholder={this.messages('boundsPlaceholder')}
-                name='osmBounds'
-                onChange={this._onChangeBounds} />
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'><Icon type='globe' /> {this.messages('osm.title')}</Panel.Title></Panel.Heading>
```

Comment:
each h3 comment from here out is gonna be a line from Abba's `Mamma Mia` maybe

Reply:
Hey no Mama Mia line